kobo/apps/stripe/tests/test_organization_usage.py

```diff
@@ -509,3 +501,36 @@ def test_get_suscription_limit_unlimited(self, usage_type):
         generate_plan_subscription(self.organization, metadata=product_metadata)
         limit = get_organization_plan_limit(self.organization, usage_type)
         assert limit == float('inf')
+
+
+@override_settings(STRIPE_ENABLED=True)
+class OrganizationsModelIntegrationTestCase(BaseTestCase):
+    fixtures = ['test_data']
+
+    def setUp(self):
+        self.someuser = User.objects.get(username='someuser')
+        self.organization = self.someuser.organization
+
+    def test_is_mmo_subscription_logic(self):
+        self.organization.mmo_override = True
+        self.organization.save()
+        assert self.organization.is_mmo is True
+
+        self.organization.mmo_override = False
+        self.organization.save()
```

Comment:
Yeah, I was probably being overly thorough here. We don't need these checks since they basically already happen in the tests in `test_organizations_api.py`. I have removed them so this test only checks the Stripe integration.